prowler/providers/aws/services/organizations/organizations_account_part_of_organizations/organizations_account_part_of_organizations.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "organizations_account_part_of_organizations",
+  "CheckTitle": "Check if account is part of an AWS Organizations",
+  "CheckType": [
+    "Logging and Monitoring"
+  ],
+  "ServiceName": "organizations",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service::account-id:organization/organization-id",
+  "Severity": "medium",
+  "ResourceType": "Other",
+  "Description": "Check if account is part of AWS Organizations.",
+  "Risk": "",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": ""
+    },
+    "Recommendation": {
+      "Text": "",
+      "Url": ""
+    }
+  },
```

Comment:
Why all these fields are empty? At least I think we can fill the risk and the remediation.

Please refer to https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Organizations/ and https://docs.bridgecrew.io/docs to find more about the remediation.